test/src/e2e.dynval/dv.n+1.test.ts

```diff
@@ -0,0 +1,152 @@
+// Copyright 2019 Kodebox, Inc.
+// This file is part of CodeChain.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import * as chai from "chai";
+import { expect } from "chai";
+import * as chaiAsPromised from "chai-as-promised";
+import * as stake from "codechain-stakeholder-sdk";
+import "mocha";
+
+import { validators as originalValidators } from "../../tendermint.dynval/constants";
+import { SDK } from "codechain-sdk";
+import { faucetAddress, faucetSecret } from "../helper/constants";
+import { PromiseExpect } from "../helper/promise";
+import { withNodes } from "./setup";
+
+chai.use(chaiAsPromised);
+
+const betty = originalValidators[0];
+const otherDynValidators = originalValidators.slice(1, 8);
+const allDynValidators = [betty, ...otherDynValidators];
```

Comment:
I think it's the same as `originalValidators`.

Reply:
Imported originalValidators has 10 members.

Reply:
Then how about changing them like this?
```js
const allDynValidators = originalValidators.slice(0, 8);
const betty = allDynValidators[0];
const otherDynValidators = allDynValidators.slice(1);
```
I think it's better to explicitly state that `allDynValidators` is a subset of `originalValidators`

Reply:
Ok, I understood it. I changed the code.